vyper/opcodes.py

```diff
@@ -52,7 +52,7 @@
     'MLOAD': [0x51, 1, 1, 3],
     'MSTORE': [0x52, 2, 0, 3],
     'MSTORE8': [0x53, 2, 0, 3],
-    'SLOAD': [0x54, 1, 1, 200],
+    'SLOAD': [0x54, 1, 1, 800],
```

Comment:
Can you add a comment to the effect of ISTANBUL RULESET

Reply:
```suggestion
    'SLOAD': [0x54, 1, 1, 800],  # Under Istanbul Ruleset
```